index.html

```diff
@@ -2255,7 +2483,7 @@ <h3 id="event-gamepadconnected">
         The <dfn class="event">gamepadconnected</dfn> event
       </h3>
       <p>
-        When a gamepad becomes available on the system, run the following
+        <dfn>When a gamepad becomes available on the system</dfn>, run the following
```

Comment:
I believe it's more correct to split this into a trigger and a process.
```suggestion
        When a [=gamepad=] <var>gamepad</var> becomes available on the system,
        [=emit `gamepadconnected` event=] for <var>gamepad</var>.
      </p>
      <p>
        To <dfn>emit `gamepadconnected` event</dfn> for given [=gamepad=] <var>gamepad</var>, run the following
```

Reply:
Also I believe the `[=current global object=]` is ambiguous, especially in case of simulated devices.